product-init/ProductInit/ProductInit/Initializer.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Bukimedia.PrestaSharp.Factories;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace ProductInit
+{
+    internal class Initializer
+    {
+        private const string SCRAPE_RESULT_PATH = @".\..\..\..\..\..\..\scrape-result";
```

Comment:
Można przemyśleć by uruchamiać to w katalogu głównym, a nie koniecznie w domyślnym, tak aby nie było tak dużo przechodzenia katalog wyżej, co jest mało czytelne

Reply:
Zmieniłem ustawienia projektu i obecnie są 2 przejścia do góry, Visual Studio tak buduje projekt, że nie wiem, czy da się to polepszyć